ci-operator/step-registry/openshift-qe/control-plane-tests/openshift-qe-control-plane-tests-workflow.yaml

```diff
@@ -0,0 +1,8 @@
+
+workflow:
+  as: openshift-qe-control-plane-tests
+  steps:
+    test:
+      - chain: openshift-qe-ocp-qe-perfscale-ci-tests
```

Comment:
should be `openshift-qe-control-plane-tests`

Reply:
Ok, updated!